rpc/src/services/protocol/proto_006/rights_service.rs

```diff
@@ -175,7 +175,12 @@ fn baking_rights_assign_rolls(parameters: &RightsParams, constants: &RightsConst
 
         // we omit the estimated_time field if the block on the requested level is already baked
         let priority_timestamp = if block_level < (level as i64) {
-            Some(estimated_head_timestamp + (priority as i64 * time_between_blocks[1]))
+            let time = if time_between_blocks.len() == 1 {

```

Comment:
@adonagy 
we should fix this in all protos

Reply:
Definitely. Will do